components/gitpod-db/src/project-db.ts

```diff
@@ -37,3 +31,12 @@ export interface ProjectDB extends TransactionalDB<ProjectDB> {
     getProjectUsage(projectId: string): Promise<ProjectUsage | undefined>;
     updateProjectUsage(projectId: string, usage: Partial<ProjectUsage>): Promise<void>;
 }
+
```

Comment:
lifted these args into an object as the ordering/number of them was getting pretty hairy, especially w/ a couple being optional.